test/NuGet.Core.Tests/NuGet.CommandLine.XPlat.Test/BasicLoggingTests.cs

```diff
@@ -69,6 +69,29 @@ public void BasicLogging_RestoreHelp_ExitCode()
         }
 
         [Fact]
+        public void BasicLogging_RestoreConfigFile_ExitCode()
+        {
+            // Arrange
+            var log = new TestLogger();
```

Comment:
There isn't a way to verify the restore parameters from xplat until after moving from dnx to dotnet cli. 

XPlat loads an assembly from DNX RC2, however the tests run under DNX RC1. When the restore method is hit a missing method exception is thrown.

Restore is currently tested at the RestoreCommand layer.